.coveragerc

```diff
@@ -1,5 +1,9 @@
 [report]
-omit = */samples/*
+omit =
+    */samples/*
+    # Don't report coverage over platform-specific modules.
+    oauth2client/contrib/_fcntl_opener.py
```

Comment:
You can merge if you're satisfied.

Timeline for multistore is today, as I'm working on it right now.

Timeline for fasteners is likely a week or so off.